tests/common/mod.rs

```diff
@@ -240,7 +241,8 @@ impl PixiControl {
         let project = self.project()?;
         let task = ExecutableTask::from_cmd_args(&project, args.task, Some(Platform::current()));
 
-        let task_env = get_task_env(&project, args.frozen, args.locked).await?;
+        let project = self.project().unwrap();
```

Comment:
Isnt the project already availabe two lines above?

Reply:
You are right that was a merge error!